dockerfiles/startall.sh

```diff
@@ -191,6 +198,21 @@ if [[ $DEPLOY_PROM == 1 ]] ; then
   helm ${CMD} myprometheus4hpccmetrics $scriptdir/../helm/$MANAGED_PROM_SUBPATH ${restArgs[@]} --set kube-prometheus-stack.prometheus.service.type=LoadBalancer --set kube-prometheus-stack.grafana.service.type=LoadBalancer
 fi
 
+if [[ $DEPLOY_LOKI == 1 ]] ; then
+  echo -e "\n\nDeploying "myloki4hpcclogs" - Grafana Loki Stack:"
+  echo -e "\n Access Grafana on: http://localhost:3000"
+  echo -e "\n user: admin"
+  echo -e "\n To obtain pass: 'kubectl get secret myloki4hpcclogs-grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo' "
+  echo -e "\n Find logs on 'Explore' view, under 'loki' datasource\n"
```

Comment:
New concept in this script, but ultimately very useful, and very low negative impact